lib/logstash/outputs/bson/big_decimal.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2009-2014 MongoDB Inc.
```

Comment:
do we need this copyright header information? not sure we have to as this is not modifying their code, right? what do you think?